src/main/java/helloworld/s3_lambda_sqs/ReadWriteS3LambdaSQS.java

```diff
@@ -4,15 +4,32 @@
 import com.amazonaws.services.lambda.runtime.LambdaLogger;
 import com.amazonaws.services.lambda.runtime.RequestHandler;
 import com.amazonaws.services.lambda.runtime.events.SQSEvent;
+import com.amazonaws.services.sqs.AmazonSQSClientBuilder;
+
+import java.util.Objects;
+
+import static helloworld.constant.Constants.QUEUE_INPUT_NAME;
+import static helloworld.constant.Constants.SQS_RECEIVED_MESSAGE;
 
 public class ReadWriteS3LambdaSQS implements RequestHandler<SQSEvent, String> {
     @Override
     public String handleRequest(SQSEvent sqsEvent, Context context) {
         LambdaLogger logger = context.getLogger();
+        Sqs queue = new Sqs(AmazonSQSClientBuilder.defaultClient(), QUEUE_INPUT_NAME);
+
+        String result = null;
 
         for (SQSEvent.SQSMessage msg : sqsEvent.getRecords()) {
-            logger.log(msg.getBody());
+            result = msg.getBody();
+            logger.log(SQS_RECEIVED_MESSAGE + msg.getBody());
         }
-        return null;
+
+        Objects.requireNonNull(result);
```

Comment:
Although this works fine here, it should be used in the context of `Fail-fast` checks, and not as a flow control. For instance, when creating an object through constructor, or when passing some parameters to your method and you want not to process further. 

Here we also assume that AWS Event won't ever come with body==null, so I'd suggest that if you want to double check, or you throw the exception from the for loop above, or checking it now for nullity.